src/handler.py

```diff
@@ -11,15 +11,14 @@
 import logging
 from smart_open import open
 import re
-from pympler import asizeof
```

Comment:
@haihongren Why was the use of `asizeof` removed? `sys.getsizeof()` returns the reference size of the data structure in but doesn't recursively look at the size of the data structure's content. `asizeof` can give a more accurate measure.

Reference: https://stackoverflow.com/a/52523163

Reply:
@William-Hill The `asizeof`  did not always properly.  My customer still ran into payload size exceeding limit error frequently with `asizeof`.  I was able to reproduce the behaviour with their log files. 

I explored various methods and found when I  converted  the log entry to string using `str(log)`, then `sys.getsizeof(str(log))`.  It worked consistently.   My customer has been using the updated code for past 8+ months and no problem reported so far.